s3-native-client/src/test/java/com/amazonaws/s3/S3NativeClientTest.java

```diff
@@ -160,6 +168,115 @@ public void onException(final CrtRuntimeException e) {
         }
     }
 
+    private class CancelTestData<T>
+    {
+        public int ExpectedPartCount;
+        public int PartCount;
+        public CompletableFuture<T> ResultFuture;
+    }
+
+    @Test
+    public void testGetObjectCancel() {
+        Assume.assumeTrue(System.getProperty("NETWORK_TESTS_DISABLED") == null);
+
+        try (final EventLoopGroup elGroup = new EventLoopGroup(9);
```

Comment:
I know it's tests, but these are some weird numbers.  Maybe make them constants since I see at least three copies.

Reply:
I shall make them constants.